pkg/pprof/cell.go

```diff
@@ -26,6 +27,15 @@ const (
 
 	// PprofPort is the flag to set the port that pprof listens on
 	PprofPort = "pprof-port"
+
+	// PprofMutexProfileFraction is the flag to enable mutex contention profiling and set the fraction of sampled events.
+	// Set to 1 to sample all events.
+	PprofMutexProfileFraction = "pprof-mutex-profile-fraction"
```

Comment:
We should probably also expose these as helm options? 

See https://github.com/cilium/cilium/blob/e336073818b6a22d60981b7b488e67d46332d68c/install/kubernetes/cilium/values.yaml.tmpl#L1723 and https://github.com/cilium/cilium/blob/e336073818b6a22d60981b7b488e67d46332d68c/install/kubernetes/cilium/templates/cilium-configmap.yaml#L860.